plugins/web-plugin/src/main/java/ru/sbtqa/tag/stepdefs/WebSteps.java

```diff
@@ -2,12 +2,15 @@
 
 public class WebSteps extends WebGenericSteps<WebSteps> {
 
-    private static WebSteps instance;
+    static final ThreadLocal<WebSteps> storage = new ThreadLocal<WebSteps>() {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this anonymous class with a call to "ThreadLocal.withInitial". ([squid:S4065](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS4065))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=126&issues=AWeIHOF_crQ7oSxgxngF&open=AWeIHOF_crQ7oSxgxngF)

[ISSUE_KEY]: AWeIHOF_crQ7oSxgxngF